package.json

```diff
@@ -9,22 +9,11 @@
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.18.3",
```

Comment:
Kindly replace these packages as they do not fall under setting up eslint.

Reply:
Ok.
Will do that.